<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSerialReg</class>
 <widget class="QDialog" name="dlgSerialReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblSerial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValidate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledSerial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ledSerial</tabstop>
  <tabstop>pbValidate</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
